packages/Select/src/Select.tsx

```diff
@@ -78,7 +78,8 @@ const Select: FunctionComponent<ISelectProps> = ({
     });
   };
 
-  const fieldName = `select-${attrs.name}`
+  const fieldName = `select-${attrs.name}`;
+  const ariaLabel = label ? undefined : defaultTextValue;
```

Comment:
const ariaLabel = label || defaultTextValue;

Reply:
Simplifying the code to const ariaLabel = label || defaultTextValue; has issue of "[object Object]" occurs because defaultTextValue might be an object. To fix this, ensure defaultTextValue is always a string.

current:   const ariaLabel = typeof label === 'string' ? label : defaultTextValue;

Reply:
defaultTextValue = 'Please Choose...',

Reply:
Can change this so 
- if `label` is truthy then don't set `aria-label`